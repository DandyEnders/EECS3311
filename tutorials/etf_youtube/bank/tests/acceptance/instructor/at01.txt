-- create a new account with id 'id1'
new("id1")
new("id2")
deposit("id1", 234)

