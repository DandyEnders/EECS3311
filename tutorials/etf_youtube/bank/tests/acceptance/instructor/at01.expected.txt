{}
->new("id1")
{Account with id id1 has balance 0}
->new("id2")
{Account with id id1 has balance 0, Account with id id2 has balance 0}
->deposit("id1",234)
{Account with id id1 has balance 234, Account with id id2 has balance 0}
